lib/node_modules/@stdlib/math/base/special/kernel-log1p/docs/repl.txt

```diff
@@ -0,0 +1,30 @@
+
+{{alias}}( f )
+    Computes log(1+f) - f for 1+f in ~[sqrt(2)/2, sqrt(2)].
```

Comment:
@kgryte , should we have backticks here around the expressions? We have them in packages such as [`log1pmx`](https://github.com/stdlib-js/stdlib/blob/develop/lib/node_modules/%40stdlib/math/base/special/log1pmx/docs/repl.txt).

Reply:
We're inconsistent on this. But probably worth putting around `log(1+f) - f` and `1+f`.